dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -117,7 +117,16 @@ protected void forkRepositoriesFound(Multimap<String, String> pathToDockerfilesI
 
                 // fork the parent if not already forked
                 if (!parentReposForked.contains(parentRepoName)) {
-                    dockerfileGitHubUtil.closeOutdatedPullRequestAndFork(parent);
+                    try {
```

Comment:
We can't extract this directly into `DockerfileGitHubUtil`. If we did, then it would work for `all` & `parent` subcommands but doesn't work `child` subcommand. 
But what we could extract this into another class `X`, where `X` inherits from `ExecutableWithNamespace` and, class `All` & class `Parent` would inherit from `X`.

Reply:
I don't think inheritance would be necessary in this scenario.

Reply:
Seems like you could use the previous implementation of `closeOutdatedPullRequestAndFork` and just check to see if it returned a fork (if it's null, it was not successful and presumably logged - you could remove from the `pathToDockerfilesInParentRepo` multimap and you would not want to add it to the `parentReposForked` list).

Reply:
In this case we would still have duplicate code.